cmake/CMakeLists.txt

```diff
@@ -68,12 +64,10 @@ else()
   if(BUILD_SHARED_PKG_LIBS)
     add_library(mpi_stubs SHARED ${MPI_SOURCES})
     set_target_properties(mpi_stubs PROPERTIES SOVERSION ${SOVERSION})
-    if(INSTALL_LIB)
-      install(TARGETS mpi_stubs LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
-    endif()
   else()
     add_library(mpi_stubs STATIC ${MPI_SOURCES})
   endif()
+  install(TARGETS mpi_stubs LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
why would we want a static version? dropping the flag, yes. leaving it when it is shared. sure, because it links to it. but static?

Reply:
Well, installing static libraries in general is useful, but installing libraries without headers don't make sense.

I had the cray systems in mind, which only allow static libs by default as a case where a user might want to install the libs.

Anyhow. there 4 cases:
 
| `liblammps`  | pkg libs | Install ? |
| ------------- | ------------- | --------------| 
| Shared  | Shared  | Yes, install all (otherwise `lmp` won't work) |
| Shared  | Static  | Install `liblammps` (for `lmp`) only |
| Static | Shared | Don't seems like a very useful combination |
| Static | Static |  (Mainly Cray) Install `liblammps.a` with header, but static pkg libs will be needed to avoid missing symbols when linking anything against `liblammps`.  |

Reply:
@rbberger: How about this - let's drop `BUILD_SHARED_PKG_LIBS` again and make them all static only (this might lead to duplicated symbol, e.g. from libatc, but let's deal with that down road when it actually happens by adding a `find_packag(ATC)`) and only install `liblammps` when build shared (so screw Cray systems for now - at least the install).

Reply:
@junghans I'm fine with dropping `BUILD_SHARED_PKG_LIBS`. All I care about is `liblammps.so` being there and its library header and not wasting any more time...